examples/src/main/java/com/mapbox/navigation/examples/core/FasterRouteActivity.kt

```diff
@@ -155,7 +155,7 @@ class FasterRouteActivity : AppCompatActivity(), OnMapReadyCallback {
             this,
             Mapbox.getAccessToken()
         )
-        mapboxNavigationOptions.onboardRouterConfig?.toBuilder()?.tilePath("")
+        mapboxNavigationOptions.onboardRouterOptions?.toBuilder()?.tilePath("")
```

Comment:
Is this intentional? What are we showcasing here?

Reply:
good catch, removing!